guide/sections/part2/global-services.adoc

```diff
@@ -202,9 +204,52 @@ wis2-gdc provides the functionality required for the Global Discovery Catalogue,
 
 wis2-gdc is managed as a free and open source project. Source code, issue tracking and discussions are hosted openly on GitHub: https://github.com/wmo-im/wis2-gdc.
 
-==== 2.7.6 Global Monitor
+==== 2.7.6 Global Replay
 
 ===== 2.7.6.1 Technical considerations
+
+* The Global Replay provides Global Services and data consumers with a mechanism to search, query and subscribe to notification messages and monitoring events of interest.
```

Comment:
The Global Replay provides Global Cache, Global Discovery Catalogue and data consumers with a mechanism to search, query and subscribe to **past** notification messages and monitoring events of interest.